tests/test_balancing_confounds.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+
+@pytest.mark.filterwarnings("ignore::sklearn.exceptions.ConvergenceWarning")
+@pytest.mark.parametrize("trained_decoder", [("tangled", "unbalancedbase"),
+                                              ("tangled", "unbalancedsvc"),
+                                              ("tangled", "unbalancednonlinear")
+                                              ], indirect=["trained_decoder"])
+def test_without_balancing(trained_decoder):
+    pval = trained_decoder.results.action.pval
```

Comment:
rename "decoder" into decodandas for consistency

Reply:
The `test_balancing_confounds` tests are now integrated into `TestDecoderCase`. See the new pull request for details.